config/config_test.go

```diff
@@ -1,48 +1,46 @@
 package config
 
 import (
-	"testing"
+    "testing"
 
-	"github.com/stretchr/testify/assert"
+    "github.com/stretchr/testify/assert"
 )
 
 func TestIsDataDogConfigValid(t *testing.T) {
-	cfg := DatadogConfig{}
+    cfg := DatadogConfig{}
 
-	assert.False(t, cfg.IsDataDogConfigValid())
+    assert.False(t, cfg.IsDataDogConfigValid())
 
-	cfg.Env = "dev"
-	assert.False(t, cfg.IsDataDogConfigValid())
+    cfg.Env = "dev"
+    assert.False(t, cfg.IsDataDogConfigValid())
 
-	cfg.Service = "Lib"
-	assert.False(t, cfg.IsDataDogConfigValid())
+    cfg.Service = "Lib"
+    assert.False(t, cfg.IsDataDogConfigValid())
 
-	cfg.ServiceVersion = "v1"
-	assert.False(t, cfg.IsDataDogConfigValid())
+    cfg.ServiceVersion = "v1"
+    assert.False(t, cfg.IsDataDogConfigValid())
 
-	cfg.DSD = "/tmp"
-	assert.False(t, cfg.IsDataDogConfigValid())
+    cfg.APM = ""
+    cfg.DSD = ""
+    assert.False(t, cfg.IsDataDogConfigValid())
 
-	cfg.DSD = "unix///tmp"
-	cfg.APM = "unix///tmp"
-	assert.False(t, cfg.IsDataDogConfigValid())
-
-	cfg.APM = "/tmp"
-	assert.True(t, cfg.IsDataDogConfigValid())
+    cfg.DSD = "unix///tmp"
+    cfg.APM = "unix///tmp"
+    assert.True(t, cfg.IsDataDogConfigValid())
```

Comment:
I think we need one more test to check if setting only one works as expected

Reply:
You men like here? https://github.com/coopnorge/go-datadog-lib/blob/main/boostrap_test.go#L16

Reply:
Just in case, since in vendor (datadog pck) now fixed the schema prefix, it's pointless to be strict as wrapper

Reply:
No more like 

```
    cfg.APM = ""
    cfg.DSD = "unix///tmp"
    assert. True(t, cfg.IsDataDogConfigValid())
```

Reply:
Updated